test/hotspot/jtreg/vmTestbase/nsk/jdi/Accessible/isPackagePrivate/accipp001.java

```diff
@@ -108,8 +108,7 @@ private int runThis (String argv[], PrintStream out) {
         logHandler      = new Log(out, argsHandler);
         Binder binder   = new Binder(argsHandler, logHandler);
 
-        debugee = Debugee.prepareDebugee(argsHandler, logHandler,
-                debugeeName + (argsHandler.verbose() ? " -vbs" : ""));
+        debugee = Debugee.prepareDebugee(argsHandler, logHandler, debugeeName);
```

Comment:
Adding -vbs to the debugeeName was messing up the changes made in prepareDebuggee(). Adding it is also redundant. If verbose is enabled, -verbose is already passed to the debuggee (-vbs and -verbose are both handled the same).